### ISSUE TITLE
Problem: Mesh agent Reconecting infinitly

### ISSUE THREAD

**Issue by Electricity2015** [Ylianst/MeshCentral#1650]
Hi 
I haveba problem ,The meshagent from 2 of my servers it connects and disconnects instantly, continuously.

The error that it give's me on log is 

started like this 
[2020-05-23 12:09:25 AM] FATAL EXCEPTION [MeshAgent_91FBC0ECD6524754.exe] @ [FuncAddr: 0x00007ff7e4623a4d / BaseAddr: 0x00007ff7e464f608 / Delta: 179131]

[2020-06-27 12:23:57 AM] FATAL EXCEPTION [MeshAgent_7E1808999D982720.exe] @ [FuncAddr: 0x00007ff629bddce9 / BaseAddr: 0x00007ff629bd7798 / Delta: -25937]

[2020-07-27 12:38:23 AM] FATAL EXCEPTION [MeshAgent_51930CE2658BCF5D.exe] @ [FuncAddr: 0x00007ff7eb6b9068 / BaseAddr: 0x00007ff7eb58b6f4 / Delta: -1235316]

[2020-07-27 12:39:25 AM] FATAL EXCEPTION [MeshAgent_51930CE2658BCF5D.exe] @ [FuncAddr: 0x00007ff7eb6b9068 / BaseAddr: 0x00007ff7eb58b6f4 / Delta: -1235316]

[2020-07-27 12:40:26 AM] FATAL EXCEPTION [MeshAgent_51930CE2658BCF5D.exe] @ [FuncAddr: 0x00007ff7eb6b9068 / BaseAddr: 0x00007ff7eb58b6f4 / Delta: -1235316]

[2020-07-27 12:41:28 AM] FATAL EXCEPTION [MeshAgent_51930CE2658BCF5D.exe] @ [FuncAddr: 0x00007ff7eb6b9068 / BaseAddr: 0x00007ff7eb58b6f4 / Delta: -1235316]

[2020-07-27 12:42:29 AM] FATAL EXCEPTION [MeshAgent_51930CE2658BCF5D.exe] @ [FuncAddr: 0x00007ff7eb6b9068 / BaseAddr: 0x00007ff7eb58b6f4 / Delta: -1235316]

[2020-07-27 12:43:31 AM] FATAL EXCEPTION [MeshAgent_51930CE2658BCF5D.exe] @ [FuncAddr: 0x00007ff7eb6b9068 / BaseAddr: 0x00007ff7eb58b6f4 / Delta: -1235316]

[2020-07-27 12:44:32 AM] FATAL EXCEPTION [MeshAgent_51930CE2658BCF5D.exe] @ [FuncAddr: 0x00007ff7eb6b9068 / BaseAddr: 0x00007ff7eb58b6f4 / Delta: -1235316]

[2020-07-27 12:45:34 AM] FATAL EXCEPTION [MeshAgent_51930CE2658BCF5D.exe] @ [FuncAddr: 0x00007ff7eb6b9068 / BaseAddr: 0x00007ff7eb58b6f4 / Delta: -1235316]

[2020-07-27 12:46:35 AM] FATAL EXCEPTION [MeshAgent_51930CE2658BCF5D.exe] @ [FuncAddr: 0x00007ff7eb6b9068 / BaseAddr: 0x00007ff7eb58b6f4 / Delta: -1235316]

and is the same eror for 12 hours

[2020-07-27 12:21:19 PM] FATAL EXCEPTION [MeshAgent_51930CE2658BCF5D.exe] @ [FuncAddr: 0x00007ff7eb6b9068 / BaseAddr: 0x00007ff7eb58b6f4 / Delta: -1235316]

**Comment by Ylianst** [comment]
Oh dear. I will try to investigate this just based the on the information above. But if you want to help, edit the meshagent.msh file and add the line `coreDumpEnabled=1`, for example:

```
MeshName=Home Computers
MeshType=2
MeshID=0xEDBE1BE3771D...EE7CD5C
ServerID=D99362D5ED8B...5135FD
MeshServer=wss://...meshcentral.com:443/agent.ashx
ignoreProxyFile=1
webSocketMaskOverride=1
coreDumpEnabled=1
```

Then, start the agent again. On the next crash, you will get a big crash dump file. It may be 20 to 30 megabytes. Send me that file, [contact info here](https://www.meshcommander.com/contact-information). We should be able to fix it right away. Thanks.

**Comment by Ylianst** [comment]
Just published MeshCentral v0.5.97 with Bryan's fix for Windows Server 2012. Update to v0.5.97 and let us know if this still happens. Thanks.
